keadm/cmd/keadm/app/cmd/util/common.go

```diff
@@ -825,6 +825,18 @@ func downloadServiceFile(componentType types.ComponentType, version semver.Versi
 		}
 		ServiceFilePath := storeDir + "/" + ServiceFileName
 		strippedVersion := fmt.Sprintf("%d.%d", version.Major, version.Minor)
+
+		// if the specified kubeedge version is greater than the latest version
+		// that means we haven't released the specified version, only occur in keadm e2e test
+		// we will use the latest version to download service file
+		latestVersion, _ := GetLatestVersion()
+		if v, err := semver.Parse(strings.TrimPrefix(latestVersion, "v")); err == nil {
+			if version.GT(v) {
+				strippedVersion = fmt.Sprintf("%d.%d", v.Major, v.Minor)
+			}
+		}
+		fmt.Printf("keadm will download version %s service file\n", strippedVersion)
```

Comment:
The `version` seems also got through `GetLatestVersion`, can we fix this issue in `Add2ToolsList` or in `GetLatestVersion`?

Reply:
seems not proper, we only use the latest version to download service file and CRDs, but we also need the version to process release package, this value should use the exact version but not the latest version.